factory/src/lib.rs

```diff
@@ -0,0 +1,189 @@
+#![no_std]
+
+use dex_factory_io::*;
+use dex_pair_io::*;
+use gstd::{exec, msg, prelude::*, prog::ProgramGenerator, ActorId};
+
+const ZERO_ID: ActorId = ActorId::zero();
+
+#[derive(Debug, Default)]
+pub struct Factory {
+    // CodeHash to deploy a pair contract from factory.
+    pub pair_code_hash: [u8; 32],
+    pub owner_id: ActorId,
+    // Who gets the fee
+    pub fee_to: ActorId,
+    pub fee_to_setter: ActorId,
```

Comment:
May be `admin` is better name for that variable

Reply:
I guess to leave it as it is, since that's the exact functionality this variable displays.